### PR TITLE
Create Indicator Description Blurbs

### PR DESCRIPTION
**Pull request by zfoteff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,26 @@ CS07: Economic Recovery Dashboard
...
-* Jessica Robertson
-* Lucas Abeln
+# About the Project
+Our group has developed a web based tool that visualizes a series of economic indicators to provide a snapshot of the Spokane economy, and other similar cities. Our project is to take this data and present it in a way that is easily digestible.  
+The project also seeks to automate the data entry as much as possible, identifying the cycle that data may be updated from sources and posted on-line and then incorporated into the dashboard. For the dashboards that are not automatically updated, an admin page allows a limited set of users to make manual updates to key indicators. 
 
-### Mentor:
-* Dan Lenz
+# Contributers:
+* Simon Watkins: swatkins@zagmail.gonzaga.edu
```
Comment (sawatkins) [inline README.md:15]:
```suggestion
* Simon Watkins: swatkins2@zagmail.gonzaga.edu
```

File: README.md [hunk 2]
```diff
@@ -40,8 +49,16 @@ After saving changes to site, restart the server.
...
-```bash
-node ./doc/swagger.js
-```
+# Usage
+
+
+## Dashboard
+
+## About Page
+The link to the [about page](https://spokaneeconomicdashboard.herokuapp.com/about) is found in the footer of the dashboard page. It includes some infomation about the project and the
```
Comment (sawatkins) [inline README.md:58]: Unfinished sentence

File: public/javascripts/indicator-config.js [hunk 3]
```diff
@@ -111,7 +111,7 @@ indicatorConfig.ndm.chartConfig = {
 // INCOME
 // median household income
 indicatorConfig.mhi = {};
-indicatorConfig.mhi.description = "Description for natural change. Description for natural change. Description for natural change. Description for natural change.Description for natural change.";
+indicatorConfig.mhi.description = "A household income is the total amount of money those living in a home earn in a year. The median household income is the income amount that divides the population into two groups: half of the homes having an income above this amount, and half of those homes having an income less than that amount. The median household income is heavily influenced by the national economy, however, there are many local factors that can influence a region's median household income.";
```
Comment (sawatkins) [inline public/javascripts/indicator-config.js:114]:
```suggestion
indicatorConfig.mhi.description = "A household income is the total amount of money those living together in the same home earn in a year. The median household income is the income amount that divides the population into two groups: half of the homes having an income above this amount, and half of those homes having an income less than that amount. The median household income is heavily influenced by the national economy, however, there are many local factors that can influence a region's median household income.";
```

Comment (sawatkins) [reply public/javascripts/indicator-config.js:114]: Just thought this was a bit more specific

**Review by sawatkins — COMMENTED** [review]
There are just a few minor suggestions that I commented

File: README.md [hunk 4]
```diff
@@ -31,7 +34,13 @@ Install pip packages
 pip3 install -r requirements.txt
```
Comment (lucasabeln) [inline README.md:34]: Should be pip for heroku app not pip3 I believe

**Review by lucasabeln — CHANGES_REQUESTED** [review]

**Review by lucasabeln — APPROVED** [review]

**Pull request merged**